README.md

```diff
@@ -24,281 +32,97 @@ of applications & services when it comes to rate limiting, authentication, autho
+Additionally, kuadrant provides the following CRDs
 
-```
-curl -v -H 'Host: api.toystore.com' -X POST http://localhost:9080/admin/toy
-```
+| CRD | Owner | Description |
+| --- | --- | --- |
+| [Kuadrant CRD](https://github.com/Kuadrant/kuadrant-operator/blob/main/api/v1beta1/kuadrant_types.go) | [Kuadrant Operator](https://github.com/Kuadrant/kuadrant-operator) | Represents an instance of kuadrant |
+| [Limitador CRD](doc/ratelimitpolicy-reference.md) | [Limitador Operator](https://github.com/Kuadrant/limitador-operator) | Represents an instance of Limitador |
+| [Authorino CRD](https://github.com/Kuadrant/authorino-operator#the-authorino-custom-resource-definition-crd) | [Authorino Operator](https://github.com/Kuadrant/authorino-operator) | Represents an instance of Authorino |
+| [AuthConfig CRD](https://github.com/Kuadrant/authorino/blob/main/docs/architecture.md#the-authorino-authconfig-custom-resource-definition-crd) | [Authorino](https://github.com/Kuadrant/authorino) | The desired authN and authZ protection for a service |
 
-Should return `200 OK` for alice
-
-```
-curl -v -H 'Host: api.toystore.com' -H 'Authorization: APIKEY ALICEKEYFORDEMO' -X POST http://localhost:9080/admin/toy
-```
+![architecture](https://i.imgur.com/0vdUZ0l.png)
 
-Should return `200 OK` for bob
+## Getting started
+
+### Pre-requisites
+
+* Istio is installed in the cluster. Otherwise, refer to the
+[Istio getting started guide](https://istio.io/latest/docs/setup/getting-started/).
+* Kubernetes Gateway API is installed in the cluster. Otherwise,
+[configure Istio to expose a service using the Kubernetes Gateway API](https://istio.io/latest/docs/tasks/traffic-management/ingress/gateway-api/).
+* Kuadrant is installed in the cluster.
+Otherwise, refer to the [kuadrant operator](https://github.com/Kuadrant/kuadrant-operator)
+for installation.
 
-```
-curl -v -H 'Host: api.toystore.com' -H 'Authorization: APIKEY BOBKEYFORDEMO' -X POST http://localhost:9080/admin/toy
-```
+### If you are an *API Provider*
 
-10. Verify authenticated ratelimit by doing `200 OK` requests 2-3 times.
+* Deploy the service/API to be protected ("Upstream")
+* Expose the service/API using the kubernetes Gateway API, ie
+[HTTPRoute](https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io/v1alpha2.HTTPRoute) object.
+* Write and apply the Kuadrant's [RateLimitPolicy](/doc/rate-limiting.md) and/or
+[AuthPolicy CRD](apis/apim/v1alpha1/authpolicy_types.go) resources targeting the HTTPRoute resource
+to have your API protected.
 
-## Demos
+### If you are a *Cluster Operator*
 
-### [Updating the RateLimitPolicy `targetRef` attribute](/doc/demo-rlp-update-targetref.md)
+* (Optionally) deploy istio ingress gateway using the
+[Gateway](https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io/v1alpha2.Gateway) resource.
+* Write and apply the Kuadrant's RateLimitPolicy custom resource targeting the Gateway resource.
```

Comment:
```suggestion
```

Reply:
I have updated that. This suggested change does not apply.

Reply:
The way I read it, it looked like the line was repeated. I guess what you want to capture here then is the difference between infrastructure RL and business level RL/auth rules maybe, both applied at the level of the Gateway?

Reply:
much simpler than that. The line was leaked. I have removed it